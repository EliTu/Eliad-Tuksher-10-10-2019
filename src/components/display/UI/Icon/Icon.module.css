.IconStyles {
    /* margin: 0 auto;
    border: 2px solid black;
    padding: 1rem;
    display: flex; */
}