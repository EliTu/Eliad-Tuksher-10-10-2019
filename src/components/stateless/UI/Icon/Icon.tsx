import React from 'react';
import styles from './Icon.module.css';

const Icon: React.FC = () => {
	const { IconStyles } = styles;

	return <div className={IconStyles}></div>;
};

export default Icon;
