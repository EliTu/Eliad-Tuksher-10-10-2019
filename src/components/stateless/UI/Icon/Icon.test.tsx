import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import Icon from './Icon';

describe('Icon component', () => {
	let component: ShallowWrapper;
	beforeEach(() => (component = shallow(<Icon />)));

	it('should render without errors', () => {
		expect(component).toMatchSnapshot();
		expect(component.length).toBe(1);
		expect(component.length).not.toBe(2);
		expect(component).toHaveClassName('IconStyles');
	});
});
