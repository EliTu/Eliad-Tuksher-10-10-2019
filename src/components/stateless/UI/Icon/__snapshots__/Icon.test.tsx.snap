// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon component should render without errors 1`] = `
<div
  className="IconStyles"
/>
`;
